metaseq/checkpoint_utils.py

```diff
@@ -113,14 +111,6 @@ def _copy_if_not_async(src, dest):
             f"Saved checkpoint {checkpoints[0]} (epoch {epoch} @ {num_update} updates) "
```

Comment:
Oh something got messed up here in the reverts somehow - had to update this to `update` instead of `num_update`.